config/crd/bases/config.nri_nriplugindeployments.yaml

```diff
@@ -14,1001 +14,1050 @@ spec:
-      served: true
-      storage: true
-      subresources:
-        status: {}
+                    type: object
+                  tolerations:
+                    description: If specified, the pod's tolerations.
+                    items:
+                      description: Toleration represents the toleration object that
+                        can be attached to a pod. The pod this Toleration is attached
+                        to tolerates any taint that matches the triple <key,value,effect>
+                        using the matching operator <operator>.
+                      properties:
+                        effect:
+                          description: Effect indicates the taint effect to match.
+                            Empty means match all taint effects. When specified, allowed
+                            values are NoSchedule, PreferNoSchedule and NoExecute.
+                            NoSchedule - do not allow new pods to schedule onto the
+                            node unless they tolerate the taint, but allow all pods
+                            submitted to Kubelet without going through the scheduler
+                            to start, and allow all already-running pods to continue
+                            running.Enforced by the scheduler. PreferNoSchedule -
+                            like TaintEffectNoSchedule, but the scheduler tries not
+                            to schedule new pods onto the node, rather than prohibiting
+                            new pods from scheduling onto the node entirely. Enforced
+                            by the scheduler. NoExecute - evict any already-running
+                            pods that do not tolerate the taint. Currently enforced
+                            by NodeController.
+                          enum:
+                          - NoSchedule
+                          - PreferNoSchedule
+                          - NoExecute
+                          type: string
+                        key:
+                          description: Key is the taint key that the toleration applies
+                            to. Empty means match all taint keys. If the key is empty,
+                            operator must be Exists; this combination means to match
+                            all values and all keys.
+                          type: string
+                        operator:
+                          description: Operator represents a key's relationship to
+                            the value. Valid operators are Exists and Equal. Defaults
+                            to Equal. Exists is equivalent to wildcard for value,
+                            so that a pod can tolerate all taints of a particular
+                            category.
+                          enum:
+                          - Exists
+                          - Equal
+                          type: string
+                        "tolerationS\teconds":
```

Comment:
That tab in `"tolerationS\teconds"` looks like a typo to me.

Reply:
I don't know how it ended up to be like that we have it in main as well https://github.com/containers/nri-plugins/blob/1f45c70f6aa07e6be68c6c18f744b5c4bc48d0cc/config/crd/bases/config.nri_nriplugindeployments.yaml#L984. I will fix it though.

Reply:
Yeah, weird. It was already in the original commit for the operator. It has been there since day 1 of the operator, already present in the original commit 4ab020681 like that. None of us spotted it then...